app/forms/journeys/additional_payments_for_teaching/eligibility_confirmed_form.rb

```diff
@@ -0,0 +1,42 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class EligibilityConfirmedForm < Form
+      attribute :selected_claim_policy, :string
+
+      validates :selected_claim_policy, presence: {
+        message: ->(object, _) { object.i18n_errors_path(:blank) }
+      }
+      validates :selected_claim_policy, inclusion: {
+        message: ->(object, _) { object.i18n_errors_path(:inclusion) },
+        in: ->(object) { object.claims_eligible_now_and_sorted.lazy.map(&:policy).map(&:to_s) }
+      }, if: -> { selected_claim_policy.present? }
+
+      delegate :eligible_now, :eligible_now_and_sorted, to: :claim, prefix: :claims
+
+      def initialize(**)
+        super
+
+        self.selected_claim_policy = permitted_params.fetch(:selected_claim_policy, session[:selected_claim_policy])
+      end
+
+      def save
+        return false unless valid?
+
+        session[:selected_claim_policy] = selected_claim_policy
```

Comment:
I feel like we're missing a table, possibly a `User/Teacher`, that we would be updating the selected claim on. Obviously that's quite a big change so this is fine!